File: 2015/02/2.py
s = 0
for p in open("input"):
    l, w, h = map(int, p.split('x'))
    d = [l, w, h]
    a = min(d)
    d.remove(a)
    b = min(d)
    s += 2*(a+b) + l*w*h
print(s)

File: 2024/02/2.jl
rs = map(l -> parse.(Int, split(l)), readlines("input"))
issafe(r) = (issorted(r) || issorted(r, rev=true)) &&
            minimum(abs, diff(r)) >= 1 &&
            maximum(abs, diff(r)) <= 3
deleteat(c, args...) = deleteat!(copy(c), args...)
ispdsafe(r) = issafe(r) ||
              any(issafe(deleteat(r, i)) for i in eachindex(r))
println(count(ispdsafe, rs))
